src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurExpenseV3ServiceImpl.java

```diff
@@ -210,6 +209,22 @@ protected ConcurWebRequest<Void> buildWebRequestForExpenseWorkflowAction(String
                 .withJsonBody(workflowDTO)
                 .build();
     }
+
+    private String buildAccountDetailMessage(ConcurEventNotificationResponse resultsDTO) {
+        if (isAccountDetailMessagesEnabled()) {
+            LOG.debug("buildAccountDetailMessage, account details messages are enabled");
+            return ConcurUtils.buildDetailMessageForWorkflowAction(resultsDTO);
+        } else {
+            LOG.debug("buildAccountDetailMessage, account details messages are disabled, returning empty string");
+            return StringUtils.EMPTY;
+        }
+    }
+    
+    private boolean isAccountDetailMessagesEnabled() {
+        boolean isDetailsEnabled = concurBatchUtilityService
```

Comment:
would it be cleaner to just return the service.methodCall?

Reply:
ok